presto-docs/src/main/sphinx/release/release-0.245.1.rst

```diff
@@ -0,0 +1,10 @@
+===============
+Release 0.245.1
+===============
+
+.. warning::
+    There is a bug causing failure at startup if function namespace manager is enabled and Thrift is not configured (:pr:`15501`).
```

Comment:
I don't think we need to repeat this warning here

Reply:
Really? As far as I can see in 0.245, this warning is still there.

Reply:
15501 is in 0.246 therefore I think we should keep the warning. @rongrong what do you think?

Reply:
I think we should keep it here

Reply:
Ya, this won't be fixed until 246. We decided to not cherry-pick the fix for 245 to make sure we'd deploy 245 ontime.